weld/subsystem/src/main/java/org/jboss/as/weld/WeldSubsystemAdd.java

```diff
@@ -110,6 +111,8 @@ protected void execute(DeploymentProcessorTarget processorTarget) {
                 // TODO add processor priority to Phase
                 processorTarget.addDeploymentProcessor(WeldExtension.SUBSYSTEM_NAME, Phase.POST_MODULE, 0x0F10, new EarApplicationScopedObserverMethodProcessor());
                 processorTarget.addDeploymentProcessor(WeldExtension.SUBSYSTEM_NAME, Phase.POST_MODULE, Phase.POST_MODULE_WELD_COMPONENT_INTEGRATION, new WeldComponentIntegrationProcessor());
+                // TODO add processor priority to Phase
+                processorTarget.addDeploymentProcessor(WeldExtension.SUBSYSTEM_NAME, Phase.POST_MODULE, Phase.POST_MODULE_ENV_ENTRY + 1, new SimpleEnvEntryCdiResourceInjectionProcessor());
```

Comment:
Should we release a WildFly core that defines a new Phase for `Phase.POST_MODULE_ENV_ENTRY +1` equivalent?

Reply:
No that does not block this.